charts/couchbase-operator/values-all.yaml

```diff
@@ -435,10 +436,12 @@ cluster:
   # maintenance mode during rebalance.
   autoscaleStabilizationPeriod:
   # -- Backup defines whether the Operator should manage automated backups, and
-  # how to lookup backup resources.
+  # how to lookup backup resources.  Refer to the documentation for supported
+  # values https://docs.couchbase.com/operator/current/howto-backup.html#enable-
+  # automated-backup
   backup:
     # -- The Backup Image to run on backup pods
-    image: couchbase/operator-backup:1.0.0
+    image: couchbase/operator-backup:1.1.0
```

Comment:
Do we want to do any processing on these based on server version?

Reply:
Not sure, what's the suggestion?)

Reply:
I think we are ok at the moment but there was a point where you had to use backup image X with server version Y I thought. We probably just need to make sure the defaults work though.